ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -945,6 +962,13 @@ Long bootStrapDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive hiveDb)
               conf.getBoolVar(REPL_EXTERNAL_WAREHOUSE_SINGLE_COPY_TASK)
                   && work.replScope.includeAllTables();
           boolean isExternalTablePresent = false;
+          boolean isSnapshotEnabed =
```

Comment:
repeated code. can this be added to a common place

Reply:
Extended a config holder class, To prevent dupe resolution of configs